src/main/java/io/vertx/cassandra/CassandraRowStream.java

```diff
@@ -28,6 +30,7 @@
   @Override
   CassandraRowStream exceptionHandler(Handler<Throwable> handler);
 
+  @GenIgnore
```

Comment:
Until codegen can expose such methods separately and generators are updated (in particular Rx and JS), the streaming feature will not be usable by alternative languages.

Not a blocker for this PR but we need to do something about it @vietj